File: API/API_UNIDADEMPRENDIMIENTO/src/Application/Api.UnidadEmprendimiento.Application/DTO`s/GEST_FORM/BancoElementoFormulario/PostBEFormularioDTO.cs

using Api.UnidadEmprendimiento.Application.DTO_s.GEST_FORM.BancoOpcResElemento;
using System.ComponentModel.DataAnnotations;


namespace Api.UnidadEmprendimiento.Application.DTO_s.GEST_FORM.BancoElementoFormulario
{
    public class PostBEFormularioDTO
    {
        [Required(ErrorMessage = "El enunciado es obligatorio")]
        [StringLength(500, ErrorMessage = "Máx 500 caracteres")]
        public string? BEFO_ENUNCIADO { get; set; }

        [Required(ErrorMessage = "Tipo de pregunta obligatorio")]
        public int TEFO_CODIGO { get; set; }
        public int BEFO_ORDEN { get; set; }
        public List<PostBORElementoDTO> BANCOOPCRESELEMENTOS { get; set; } = new();

    }
}

File: FRONTEND/UNIDADEMPRENDIMIENTO/src/app/core/models/ELEMENTSFORM/PostBEFormularioDTO.ts

import { PostBORElementoDTO } from "./PostBORElementosDTO";
import { TipoElemento } from "./TipoElemento";

export interface PostBEFormularioDTO
{
  BEFO_ENUNCIADO: string;
  TEFO_CODIGO: TipoElemento | number;
  BEFO_ORDEN?: number;
  BANCOOPCRESELEMENTOS?: PostBORElementoDTO[];
}